src/textual/app.py

```diff
@@ -758,6 +758,7 @@ async def process_messages(self) -> None:
 
             driver = self._driver = self.driver_class(self.console, self)
             driver.start_application_mode()
+            driver.enable_bracketed_paste()
```

Comment:
Would it make sense to do enable the bracketed paste mode as part of `start_application_mode` ?

Reply:
I think if I would do that, I'd probably want to rename that method as they're distinct things in my head. I'll leave it like this for now